packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/import_already_converted.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('hautelook_alice', [
+        'fixtures_path' => 'fixtures',
+    ]);
+};
```

Comment:
This file is only here. The fixtures are copied to temp directory. To make this file available also in fixtures directory, it must be copied in a test case. See the other `Filesystem` calls there, for the same reason :)